src/main/java/com/ddudu/goal/domain/Goal.java

```diff
@@ -89,6 +89,15 @@ public void applyGoalUpdates(
     this.privacyType = isNull(privacyType) ? DEFAULT_PRIVACY_TYPE : privacyType;
   }
 
+  public boolean isCreatedByUser(Long userId) {
+    if (userId == null || this.user == null) {
+      return false;
+    }
+
+    return this.user.getId()
+        .equals(userId);
```

Comment:
```suggestion
    return Objects.deepEquals(this.user.getId(), userId);
```
이 부분 한 줄로 바꿀 수 있는데, 어떻게 생각하세요?

Reply:
헉 좋습니당! 그런데 배열이 아니면 `Object.equals()`와 `Object.deepEquals()`가 같은 기능을 하는 것 같은데,
` deepEquals()`를 선택하신 이유가 있나요? 단순히 궁금해서 질문드립니당..!

Reply:
앗 죄송합니다!!! equals가 두 파리미터를 모두 null check를 하는게 아니라 하나만 한다고 착각했었네요..`Objects.equals()`가 나은 것 같습니다!!!!